modules/assignment/src/main/java/com/intuit/wasabi/assignment/cache/impl/AssignmentsMetadataCacheRefreshTask.java

```diff
@@ -49,7 +49,8 @@ public AssignmentsMetadataCacheRefreshTask(AssignmentsMetadataCache metadataCach
     @Override
     public void run() {
         try {
-            LOGGER.info("AssignmentsMetadataCache refresh started at = {}", timeService.getCurrentTime());
+            Date startTime = timeService.getCurrentTime();
+            LOGGER.debug("AssignmentsMetadataCache refresh started at = {}", startTime);
```

Comment:
why LOGGER.debug() and then using info() later at line 65 when logging the total time it takes to refresh?